src/workspaces/common/state/useWorkspace.ts

```diff
@@ -115,7 +115,7 @@ export const useWorkspace = (namespace, name): WorkspaceDetails => {
       if (responseContainsRequesterPaysError(errorText)) {
         // loadGoogleBucketLocation will not get called in this case because checkBucketReadAccess fails first,
         // but it would also fail with the requester pays error.
-        setGoogleStorage({ fetchedLocation: 'ERROR', location, locationType });
+        setGoogleStorage({ fetchedLocation: 'RPERROR', location, locationType });
```

Comment:
since this is where we inspect the error to find requester-pays problems (line 115), this is where we can set `RPERROR`